rights.json

```
@@ -0,0 +1,203 @@
+    "http://creativecommons.org/licenses/by-sa/3.0/au/": {"P275": "Q86239208"},
+    "http://creativecommons.org/licenses/by-sa/3.0/ee/": {"P275": "Q86239559"},
+    "http://creativecommons.org/licenses/by-sa/3.0/es/": {"P275": "Q86239991"},
+    "http://creativecommons.org/licenses/by-sa/3.0/fr/": {"P275": "Q86240326"},
+    "http://creativecommons.org/licenses/by-sa/3.0/lu/": {"P275": "Q86240624"},
+    "http://creativecommons.org/licenses/by-sa/3.0/ro/": {"P275": "Q86241082"},
+    "http://creativecommons.org/licenses/by-nc-sa/2.0/fr/": {"P275": "Q94507369"},
+    "http://creativecommons.org/licenses/by-nc-sa/2.5/ch/deed.en": {"P275": "Q96473808"},
+    "http://creativecommons.org/licenses/by-sa/3.0/it/": {"P275": "Q98755364"},
+    "http://creativecommons.org/licenses/by-sa/3.0/tw/": {"P275": "Q98960995"},
+    "http://creativecommons.org/licenses/by-sa/3.0/nl/": {"P275": "Q18195572"},
+    "http://creativecommons.org/licenses/by-sa/3.0/de/": {"P275": "Q42716613"},
```

Biome (2.4.15)

[error] The key http://creativecommons.org/licenses/by-sa/3.0/de/ was already declared.

(lint/suspicious/noDuplicateObjectKeys)